domains/amc.json

```
@@ -0,0 +1,12 @@
+{
+    "description": "It is a portfolio",
+    "repo": "https://github.com/AdrianMartinCano/portfolio/",
+    
+    "owner": {
+        "username": "AdrianMartinCano",
+        "email": "[amc.1994.mca@gmail.com]"
+    },
+    "record": {
+        "CNAME": "AdrianMartinCano.github.io"
```

cannot access website, 404 not found
if the records are correct then provide a preview/link of your website
```suggestion
        "CNAME": "adrianmartincano.github.io"
```